### DISCUSSION TITLE
Upgade to v2

### DISCUSSION THREAD

**Discussion by davidwhicker** [westacks/telebot#62] [Q&A] [answered]
Sorry to trouble you. I have upgraded to v2, but hitting error when running command such as setup webhook.

Error:  Class `WeStacks\TeleBot\Interfaces\UpdateHandler` not found

Any help would be appreciated. 

```php
<?php
namespace App\TelBot;

use App\Http\Controllers\UserGateController;
use Illuminate\Support\Facades\Log;
use WeStacks\TeleBot\Interfaces\UpdateHandler;
use WeStacks\TeleBot\Objects\Update;
use WeStacks\TeleBot\TeleBot;

class Gateway extends UpdateHandler
{
    public static function trigger(Update $update, TeleBot $bot): bool
    {
        return isset($update->message); // handle regular messages (example)
    }

    public function handle()
    {
        $userGate = new UserGateController();

        // Log User who starts the Bot
        $userGate->saveUser($this->update);

        // Is User Allowed To Access The Bot?
        if ($userGate->isBotAccessAllowed($this->update) == false) {
            // Stop future commands from processing
            $userGate->denyUserAlertAdmin($this->update);

            // Kill future commands
            exit();
        }
    }
}
```

**Comment by punyflash** [comment, marked as answer]
In [v2](https://github.com/westacks/telebot/blob/master/CHANGELOG.md#200---2022-06-06), classes:
- `WeStacks\TeleBot\Interfaces\CommandHandler`
- `WeStacks\TeleBot\Interfaces\UpdateHandler`

changed their namespaces to:

- `WeStacks\TeleBot\Handlers\CommandHandler`
- `WeStacks\TeleBot\Handlers\UpdateHandler`

**Comment by davidwhicker** [comment]
Thank you. Ill test this out. Is it possible to update the docs here: https://westacks.github.io/telebot/#/updates?id=creating-handlers with the new namespaces.

**Reply by punyflash** [reply]
Docs are written according to latest version, there is nothing to change. If you find a mistake in the docs, I'll gladly accept a pull request.

**Comment by davidwhicker** [comment]
https://westacks.github.io/telebot/#/updates?id=creating-handlers

Still has: use WeStacks\TeleBot\Interfaces\UpdateHandler;

I dont know the full fix or I would do a PR.

**Reply by punyflash** [reply]
I see now. Ok, I'll fix doc with patch for bot api 6.4

**Comment by davidwhicker** [comment]
Thank you. Kind regardsDavid WhickerOn 31 Dec 2022, at 07:41, Dmytro Morozov ***@***.***> wrote:﻿
I see now. Ok, I'll fix doc with patch for bit api 6.4
